### DISCUSSION TITLE
End_Source For Music

### DISCUSSION THREAD

**Discussion by lowypan** [diesel-modding/BeardLib-Editor#762] [Ideas]
There is already a start_source
but some original game tracks like "Black Light" have some type of Fade Out

so, we need that.

**Comment by Luffyyy** [comment]
Recommend it here instead https://github.com/simon-wh/PAYDAY-2-BeardLib since this is just the editor and has nothing to do with loading content.

**Reply by lowypan** [reply]
ups
